gdpr-samples/gdpr-qsg/src/main/resources/gdpr.sh

```diff
@@ -499,6 +498,14 @@ curl -s -k -H "Authorization: Basic ${auth}" -H "Content-Type: text/xml" -H "SOA
             <xsd1:permissionAndRoleConfig></xsd1:permissionAndRoleConfig>
             <!--Optional:-->
             <xsd1:saasApp>false</xsd1:saasApp>
+            <xsd1:owner>
```

Comment:
I was under the impression we were not going to use the default admin user in any of our demonstrations of the QSG

Reply:
Yes this should be Cameron

Reply:
Is there any specific reason for that?

Reply:
An issue https://github.com/wso2/samples-is/issues/80 created will be fix this for the next release